app/services/idv/profile_maker.rb

```diff
@@ -26,6 +26,13 @@ def save_profile(
       profile.encrypt_pii(pii_attributes, user_password)
       profile.proofing_components = current_proofing_components
       profile.fraud_pending_reason = fraud_pending_reason
+
+      if in_person_verification_needed
+        profile.idv_level = :legacy_in_person
+      else
+        profile.idv_level = :legacy_unsupervised
+      end
```

Comment:
my personal preference is to assign from the result of if statements:
```suggestion

      profile.idv_level = if in_person_verification_needed
         :legacy_in_person
      else
        :legacy_unsupervised
      end
```